README.md

```diff
@@ -41,4 +41,10 @@ if err != nil {
 
 log.Println(string(v))
 // => Practice
+
+// Iterate over the database
+db.Each(func(key, value []byte) error {
```

Comment:
Got some time to look at it now actually. Do you want the iterator type defined in its own file or in cdb.go?